pkgs/applications/science/medicine/dcmtk/0001-Fix-cmake.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/CMake/dcmtk.pc.in b/CMake/dcmtk.pc.in
+index 13c79c0d5..b1edf725c 100644
+--- a/CMake/dcmtk.pc.in
++++ b/CMake/dcmtk.pc.in
+@@ -1,6 +1,6 @@
+  prefix="@CMAKE_INSTALL_PREFIX@"
+  exec_prefix="${prefix}"
+- libdir="${prefix}/@CMAKE_INSTALL_LIBDIR@"
++ libdir=@CMAKE_INSTALL_FULL_LIBDIR@"
```

Comment:
The upstream PR has two hunks (https://github.com/DCMTK/dcmtk/pull/75/files), does this really work when only having one hunk?